src/Our.Umbraco.Ditto/Extensions/TypeInferenceExtensions.cs

```diff
@@ -8,7 +10,7 @@
     /// Extensions methods for <see cref="T:System.Type"/> for inferring type properties.
     /// Most of this code was adapted from the Entity Framework
     /// </summary>
-    internal static class TypeInferenceExtensions
+    public static class TypeInferenceExtensions
```

Comment:
General thinking is that we don't expose too much (internals) from the Ditto core.
I'd hate for us to refactor code in future that other developers _might_ come to use/rely on.